gobblin-api/src/main/java/gobblin/publisher/DataPublisher.java

```diff
@@ -54,8 +82,7 @@ public abstract void publishMetadata(Collection<? extends WorkUnitState> tasks)
    * @param states is a {@link Collection} of {@link WorkUnitState}s.
    * @throws IOException if there is a problem with publishing the metadata or the data.
    */
-  public void publish(Collection<? extends WorkUnitState> states)
-      throws IOException {
+  public void publish(Collection<? extends WorkUnitState> states) throws IOException {
     publishMetadata(states);
```

Comment:
The javadocs for publishMetadata() say "Examples of publishing metadata include writing offset files, checkpoint files, etc.". We need to very precise what constitutes metadata as it has important consequences. Which are these implementations and what metadata is being published?